src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -415,4 +434,39 @@ std::pair<std::string, mediapipe_packet_type_enum> MediapipeGraphDefinition::get
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "setting input stream: {} packet type: {} from: {}", "", "UNKNOWN", streamFullName);
     return {"", mediapipe_packet_type_enum::UNKNOWN};
 }
+
+Status MediapipeGraphDefinition::initializeNodes() {
```

Comment:
What happens when we reload config.json but graph is in use?
- does re-loading re-initialize the nodes, meaning OvmsPythonModel object is re-created?
- if so, what if at least one request is still processing python script?
- do we have unit test for that?

Reply:
It re initializes the nodes. Same logic as with mediapipe graphs.

Reply:
Unit tests can be added once request handling is implemented.